pkg/repository/v1/sqlcv1/payload-store.sql

```diff
@@ -104,16 +104,13 @@ WITH tenants AS (
 
 SELECT *
 FROM v1_payload_wal
-WHERE
-    offload_at < NOW()
-    AND tenant_id = ANY(SELECT tenant_id FROM tenants)
-ORDER BY offload_at, payload_id, payload_inserted_at, payload_type, tenant_id
-FOR UPDATE
+WHERE tenant_id = ANY(SELECT tenant_id FROM tenants)
+ORDER BY offload_at, tenant_id, payload_id, payload_inserted_at, payload_type
```

Comment:
I think that the mis-ordering of this `ORDER BY` was causing really poor performance on this query